src/instructlab/sdg/generate_data.py

```diff
@@ -295,6 +315,7 @@ def generate_data(
     batch_size: Optional[int] = None,
     checkpoint_dir: Optional[str] = None,
     max_num_tokens: Optional[int] = DEFAULT_MAX_NUM_TOKENS,
+    docling_model_path: Optional[str] = None,
```

Comment:
no, users don't need to know about this, it will get propogated as `None` until the last step where there is a check for `none` which triggers a download from HF

Reply:
Hmm, let me rephrase. Do we expect anyone to ever pass in a path here, bypassing our logic to automatically find the path? Or is it just an oversight that we're adding this as an additional parameter a user could pass in the `generate_data`?